openstack/glance/templates/etc/_ratelimit.yaml.tpl

```diff
@@ -0,0 +1,28 @@
+rates:
+  # global rate limits counted across all projects
+  global:
```

Comment:
`global` rate limits: all requests that pass this middleware. independent of scope (initator_project_id). meant to protect glance from receiving more traffic than it can handle